src/styles/functions.scss

```diff
@@ -0,0 +1,3 @@
+@function center-position($elementSize) {
+  @return ($stopwatch-size / 2) - $elementSize;
```

Comment:
The variable `$stopwatch-size` is used in the calculation but is not defined within this file. Ensure that `$stopwatch-size` is defined globally or imported from another SCSS file before using it in this function.